Directory.Build.targets

```diff
@@ -1,4 +1,5 @@
 <Project>
+  <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
```

Comment:
After importing arcade we might be able to remove the workaround to translate the url for source link since that is already done in Arcade.